database/gormdb_assignment.go

```diff
@@ -145,8 +148,8 @@ func (db *GormDB) CreateBenchmark(query *pb.GradingBenchmark) error {
 
 // UpdateBenchmark updates the given benchmark
 func (db *GormDB) UpdateBenchmark(query *pb.GradingBenchmark) error {
-	return db.conn.Model(query).
-		Where(&pb.GradingBenchmark{ID: query.ID, AssignmentID: query.AssignmentID}).
+	return db.conn.
+		Where(&pb.GradingBenchmark{ID: query.ID, AssignmentID: query.AssignmentID, ReviewID: query.ReviewID}).
```

Comment:
Here we should probably move the different fields to separate lines.

Reply:
And done.